maestro-sdk.cabal

```diff
@@ -49,6 +49,7 @@ library
       Maestro.API.V1.Addresses
       Maestro.API.V1.Blocks
       Maestro.API.V1.Datum
+      Maestro.API.V1.DefiMarkets
```

Comment:
I would usually prefer just "Markets" as it matches exactly with path name but of course not a big deal.